modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,217 @@
+read.
++
+Mule also enables content enrichment of messages (through
+xref:target-variables[Target Parameters]) so that you can retrieve
+additional data and attach it to the message.
+
+Note that you can also create your own Event processors using the
+xref:1.1@mule-sdk::index.adoc[Mule SDK].
+
+For more on this topic, see xref:about-components[Core Components] 
+and xref:connectors:index.adoc[Connectors and Modules].
+
+//image::flow1-1.png[Flow1-1]
+
+=== Anypoint Studio and Design Center
+
+You can design Mule apps in Anypoint Studio, in Design Center, or even with
+handwritten XML. Studio and Design Center UIs represent message processors as
+icons within flow components. Studio also represents them as XML elements in a
+Configuration XML.
+
+////
+The example below is a simple flow created in Anypoint Studio's visual editor;
+click the *XML Editor* tab to view the XML of a Mule project that holds the same
+flow.
+
+[.ex]
+=====
+[discrete.view]
+=== Studio Visual Editor
+
+image::simple-flow2.png[simple+flow2]
+
+[discrete.view]
+=== XML Editor
+
+[source,xml,linenums]
+----
+include::{examplesdir}/mule-concepts.xml[]
+----
+=====
+////
+
+For more on this topic, see
+xref:design-center/about-designing-a-mule-application.adoc[Mule Flow Design (Design Center)]
+and xref:7.2@studio:index.adoc[Anypoint Studio].
+
+[[dataweave]]
+== DataWeave
+
+DataWeave is the primary language used for formulating expressions in Mule.
+Within fields of many message processing components, in addition to the
+Transform Message component, you can use DataWeave to access, manipulate, 
+transform, and extract data from the Mule event.
+
+At runtime, Mule evaluates expressions while executing a flow to:
+
+* Extract data needed to process the current message.
+* Set or manipulate a value in the message.
+
+For more on this topic, see xref:dataweave.adoc[DataWeave Language].
+
+== Batch Processing
+
+Mule supports batch jobs, which split messages into individual records, 
+act on each record, then report the results and, if necessary, push the processed 
+output to other systems or queues. This functionality is particularly useful when 
+working with streaming input or when engineering "near real-time" data integration 
+between SaaS applications.
+
+Studio and Design Center provide various batch components that you can use to 
+configure batch processes. For more on this topic, see 
+xref:batch-processing-concept.adoc[Batch Processors].
+
+== Non-blocking Execution Engine
+
+The Mule execution engine is based on a non-blocking runtime. This is a
+task-oriented execution model that allows you to take advantage of non-blocking
+IO calls and to avoid performance problems.
+
+Flows always function synchronously. If you need to achieve asynchronous
+patterns, such as Fire-and-Forget, you can use the Async (`<async>`) component.
+
+Mule Event processors can indicate to the runtime whether they are CPU intensive,
```

Comment:
remove `can`, all of them do (only difference is whether is explicit or inferred)